### ISSUE TITLE
Arguments not used - mini warning fix

### ISSUE THREAD

**Issue by poppyseedDev** [allemanfredi/suffragium-contracts#1]
https://github.com/allemanfredi/suffragium/blob/develop/contracts/contracts/IdentityManager.sol#L26
One of the arguments do not seem used.

**Comment by allemanfredi** [comment]
Yes, that's correct. `identityPublicValues` and `identityProofBytes` will be placed within `verifyProof` as a replacement for `abi.encodePacked("")`. They are not used to facilitate testing.
